TodoApp/Tests/TodoApp.TestContracts/Extensions/ItemComparer.cs

```diff
@@ -2,12 +2,17 @@
 using System.Collections.Generic;
 using TodoApp.Contracts.Models;
 
-namespace TodoApp.Api.Tests.Extensions
+namespace TodoApp.TestContracts.Extensions
 {
     public class ItemComparer : IEqualityComparer<Item>
```

Comment:
Make the class internal - it will make comparer-related code shared "by design".

Reply:
Move the class to a stand-alone folder now, since it is being used by both `Extensions` and `Wrappers`…